README.md

```diff
@@ -1,10 +1,36 @@
-# Neovim client for julia
+# Neovim client for Julia
 
 [![Build Status](https://travis-ci.org/bfredl/Neovim.jl.svg?branch=master)](https://travis-ci.org/bfredl/Neovim.jl)
 
-This is a neovim api client for julia. It supports embedding a nvim process in julia and conversely acting as a child process to nvim, as well as connecting to an external instance over a socket. It also works as a plugin host. Currently it assumes `nvim` is in `$PATH`
+Neovim.jl is a Neovim API client and plugin host for Julia. It supports:
 
-Simplest way to test the api client is to spawn an embedded instance:
+- Embedding a nvim process in Julia
+- Acting as a child process to nvim
+- Connecting to external instances over a socket
+
+Currently it assumes `nvim` is in `$PATH`.
+
+
+## Requirements
+
+- Julia 1.0
+- Neovim 0.4 <!-- FIXME: is this right? -->
+
+
+## Installation
+
+Add this package to your current Julia environment:
+```julia
+using Pkg
+Pkg.add(url="https://github.com/bfredl/Neovim.jl")
+```
+
+
+## Usage
+
+### Using as an embedded process
```

Comment:
Just a thought, perhaps the `Using as` part of these subheadings can be dropped, since they're under the `Usage` heading? E.g. this would be `As an embedded process`, and the one lower down `As a plugin host`.